setup.py

```diff
@@ -84,6 +84,7 @@
         "PyYAML==6.0",
         "six>=1.15.0",
         "tenseal==0.3.0",
+        "docker>=6.0",
```

Comment:
no question regarding this change.

Just a general feedback, we really need to sort out the dependencies with a better package strategy.

Reply:
Agreed.  This requires some improvement.